js/loginbuttonlogic.js

```diff
@@ -1,5 +1,17 @@
-function openLogin() {
-    console.log(document.getElementById("overlay"));
-    document.getElementById("overlay").style.visibility= 'visible';
-    document.getElementById("logInModal").style.visibility= 'visible';
+const logInButton = document.querySelector('.logInButton'),
+    overlay = document.querySelector('.overlay'),
+    logInModal = document.querySelector('.modal'),
+    closeLogInModalButton = document.querySelector('.close-modal');
+
+function openLoginModal() {
+    overlay.classList.add('visible');
+    logInModal.classList.add('visible');
 }
+
+function closeLoginModal () {
+    overlay.classList.remove('visible');
+    logInModal.classList.remove('visible');
+}
+
+logInButton.addEventListener('click', openLoginModal);
```

Comment:
`addEventListener` is, in my opinion, the best way for you, right now, to handle [event binding](https://developer.mozilla.org/en-US/docs/Web/Events/Event_handlers#eventtarget.addeventlistener). this way, you don't have to have any JS mixed into your HTML.